tests/test_operators.py

```diff
@@ -207,3 +210,51 @@ def test_shares_no_elements_with(self):
                          shares_no_elements_with([2, 3]))
         self.assertFalse(SelectMultipleType([1, 2, "a"]).
                          shares_no_elements_with([4, "A"]))
+
+
+class DateOperatorTests(TestCase):
+
+    def test_equals_to(self):
+        self.assertTrue(DateType('10-10-2019').
+                        equal_to('10-10-2019'))
+        self.assertFalse(DateType('10-10-2019').
+                         equal_to('10-11-2019'))
+
+    def test_not_equals_to(self):
+        self.assertTrue(DateType('10-10-2019').
+                        not_equal_to('10-11-2019'))
+        self.assertFalse(DateType('10-10-2019').
+                         not_equal_to('10-10-2019'))
+
+    def test_greater_than(self):
+        self.assertTrue(DateType('10-10-2019').
+                        greater_than('10-9-2019'))
+        self.assertFalse(DateType('10-10-2019').
+                         greater_than('10-11-2019'))
+
+    def test_greater_than_or_equal_to(self):
+        self.assertTrue(DateType('10-10-2019').
+                        greater_than_or_equal_to('10-10-2019'))
+        self.assertFalse(DateType('10-9-2019').
+                         greater_than_or_equal_to('10-10-2019'))
+        self.assertTrue(DateType('10-11-2019').
+                        greater_than_or_equal_to('10-10-2019'))
+        self.assertFalse(DateType('10-9-2019').
+                         greater_than_or_equal_to('10-10-2019'))
+
+    def test_less_than(self):
+        self.assertTrue(DateType('10-9-2019').
+                        less_than('10-10-2019'))
+        self.assertFalse(DateType('10-10-2019').
+                         less_than('10-9-2019'))
+
+    def test_less_than_or_equal_to(self):
+        self.assertTrue(DateType('10-10-2019').
+                        less_than_or_equal_to('10-10-2019'))
+        self.assertFalse(DateType('10-10-2019').
```

Comment:
i'd say - not needed
the last one covers this case and is consistent with other tests